inst/components/scss/sidebar.scss

```
@@ -54,14 +64,17 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--bslib-sidebar-icon-size)),
     border-right: var(--bslib-sidebar-vert-border);
     border-top-right-radius: 0;
     border-bottom-right-radius: 0;
-    background-color: var(--bslib-sidebar-bg);
     color: var(--bslib-sidebar-fg);
+    background-color: var(--bslib-sidebar-bg);
```

For sake of a minimal diff

```suggestion
    background-color: var(--bslib-sidebar-bg);
    color: var(--bslib-sidebar-fg);
```

---

If we drop `backdrop-filter` then yeah, I'll swap this back